components/private/Color/src/UIColor+MaterialDynamic.m

```diff
@@ -36,4 +36,16 @@ + (UIColor *)colorWithUserInterfaceStyleDarkColor:(UIColor *)darkColor
 #endif
 }
 
+- (UIColor *)mdc_resolvedColorWithTraitCollection:(UITraitCollection *)traitCollection {
+#if defined(__IPHONE_13_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0)
+  if (@available(iOS 13.0, *)) {
+    return [self resolvedColorWithTraitCollection:traitCollection];
```

Comment:
Can we move the `#if defined(__IPHONE_13_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0)` check into the `if (@available(iOS 13.0, *))` conditional?